### DISCUSSION TITLE
3 Digits (2.523) on Nova Number or Currency field

### DISCUSSION THREAD

**Discussion by StanBarrows** [laravel/nova-issues#5250] [Q&A] [answered]
I thought setting the step to 0.001 or 'any' allows three digits. But after saving the input, it applies a rounding.  Any Idea how to get this work?

```
         Number::make(__('backend/fields.multiplicator'), 'multiplicator')
                ->sortable()
                ->step('any')
                ->rules('nullable', 'required_without_all:price_per_unit', 'numeric', )

```

<img width="1231" alt="CleanShot 2023-01-19 at 16 52 25@2x" src="https://user-images.githubusercontent.com/10268813/213489416-45940d37-bf42-4dcb-b03d-9cc6c72398ad.png">

<img width="1405" alt="CleanShot 2023-01-19 at 16 52 37@2x" src="https://user-images.githubusercontent.com/10268813/213489455-43e48078-6b45-4ddd-bfee-41ea273b642d.png">

Best regards
Sebastian

**Comment by crynobone** [comment]
You can't use the default Currency field for this. In order to avoid rounding issue we also check the step against currency minimum step (default to USD)

**Comment by StanBarrows** [comment]
Many thx for the feedback

But the multiplicator field is a number field.

![CleanShot 2023-01-21 at 10 52 36@2x](https://user-images.githubusercontent.com/10268813/213861619-d68ccf99-57b4-4ae4-b117-54c2aeacdf65.png)

**Comment by StanBarrows** [comment, marked as answer]
Problem solved, the default Database setting for float is (8,2).

`$table->float('multiplicator', 8, 4)->nullable();`
